components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/autofill/GeckoLoginDelegateWrapper.kt

```diff
@@ -20,10 +20,8 @@ import org.mozilla.geckoview.GeckoResult
  * them to [storageDelegate]. This allows us to avoid duplicating [LoginStorageDelegate] code
  * between different versions of GeckoView, by duplicating this wrapper instead.
  */
-@Suppress("Deprecation")
-// This will be addressed in https://github.com/mozilla-mobile/android-components/issues/10093
 class GeckoLoginDelegateWrapper(private val storageDelegate: LoginStorageDelegate) :
```

Comment:
I am wondering if we should add an issue to remove this. It is only used in FirefoxReality https://searchfox.org/mozilla-mobile/search?q=GeckoLoginDelegateWrapper&path=

Reply:
Yeah, we can! We can also close the Fenix one, right? https://github.com/mozilla-mobile/fenix/issues/19080